evm.md

```diff
@@ -1236,8 +1234,8 @@ The various `CALL*` (and other inter-contract control flow) operations will be d
 -   The `callLog` is used to store the `CALL*`/`CREATE` operations so that we can compare them against the test-set.
 
 ```k
-    syntax Call ::= "{" Int "|" Int "|" Int "|" WordStack "}"
- // ---------------------------------------------------------
+    syntax Call ::= "{" Int "|" Int "|" Int "|" WordStack "}" [unused]
+ // ------------------------------------------------------------------
```

Comment:
Let's not include _any_ `unused` attributes until the discussion in https://github.com/kframework/k/issues/1377 is resolved.

Reply:
Actually, this looks actually unused. Can you just remove it?